### PR TITLE
issue-270: Moved Customer Edit button under Profile

### LINKED ISSUE #158: Main_Profile
- [x] getCustomerProfile
- [x] editCustomerProfile
- [x] getAppoinments (possible duplicate)
- [x] getRequests (possible duplicate)
- [ ] getBalances (possible duplicate)

### LINKED ISSUE #270: Move Edit profile button under profile view page

### PR DESCRIPTION
**Pull request by eunbeek**

### CODE REVIEW

**Review by rko7 — APPROVED** [review]

File: src/Customer/CustomerProfileEdit.js [hunk 1]
```diff
@@ -84,24 +82,13 @@ class CustomerProfileEdit extends React.Component {
...
   render() {
-    if (this.state.completed) {
-      return (
-        <Redirect
-          push
-          to={{
-            pathname: `/Customer/${this.props.id}`,
-          }}
-        />
-      );
```
Comment (yuanLeeMidori) [inline src/Customer/CustomerProfileEdit.js:95]: why are these removed? Isn't this make the save after edit staying the form page instead of going back to the customer detail page?

Comment (eunbeek) [reply src/Customer/CustomerProfileEdit.js:95]: I used refresh function to reload all data in same page to check the data is changed successfully

**Review by sonechca — APPROVED** [review]
